### DISCUSSION TITLE
Should we move the "colour.implementation" sub-package in its own repository?

### DISCUSSION THREAD

**Discussion by thomasmansencal** [colour-science/colour#689] [Ideas]
@MichaelMauderer I was wondering if the **colour.implementation** sub-package should be moved into a separate repository on its own?

**Comment by MichaelMauderer** [comment]
Where you thinking of one repository for the whole package or different ones for each sub-package?

It might be good for the visibility of the sub-packages to have a separate one each, so if people are looking for something specific they can easily find it without having to dig through a lot of unrelated stuff. (Especially if the sub-packages are completely unrelated)

**Comment by thomasmansencal** [comment]
I was thinking about the sub-package as a whole but your suggestion is excellent!
